28-08-2021 10:30:23 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 14:49:35 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 14:49:35 - [INFO] -  exploration - (exploration.py).countplots(line 16) - countplots successfully.
28-08-2021 14:49:36 - [INFO] -  exploration - (exploration.py).violinplots(line 26) - violinplots successfully
28-08-2021 14:49:37 - [INFO] -  exploration - (exploration.py).violinplots(line 26) - violinplots successfully
28-08-2021 14:49:37 - [INFO] -  exploration - (exploration.py).violinplots(line 26) - violinplots successfully
28-08-2021 14:49:38 - [INFO] -  exploration - (exploration.py).boxplots(line 42) - boxplots successfully
28-08-2021 14:49:39 - [INFO] -  exploration - (exploration.py).jointplots(line 48) - jointplots successfully
28-08-2021 14:49:42 - [INFO] -  exploration - (exploration.py).pairplots(line 59) - pairplots successfully
28-08-2021 14:49:45 - [INFO] -  exploration - (exploration.py).swarmplots(line 67) - swarmplots successfully
28-08-2021 14:49:51 - [INFO] -  exploration - (exploration.py).swarmplots(line 67) - swarmplots successfully
28-08-2021 14:49:55 - [INFO] -  exploration - (exploration.py).swarmplots(line 67) - swarmplots successfully
28-08-2021 14:49:56 - [INFO] -  exploration - (exploration.py).correlation(line 74) - plots correlation map successfully
28-08-2021 14:49:59 - [INFO] -  exploration - (exploration.py).correlation(line 74) - plots correlation map successfully
28-08-2021 15:03:41 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 15:05:20 - [ERROR] -  structure - (structure.py).drawgraph(line 17) - plots causal graph  failed.
Traceback (most recent call last):
  File "/home/neba/Desktop/Industry - Casualty Challenge/scripts/structure.py", line 12, in drawgraph
    viz = plot_structure( sm, graph_attributes={"scale": "2.0","size": "2.5"}, all_node_attributes=NODE_STYLE.WEAK, all_edge_attributes=EDGE_STYLE.WEAK)
  File "/home/neba/.conda/envs/feature/lib/python3.7/site-packages/causalnex/plots/plots.py", line 105, in plot_structure
    sm, all_node_attributes, all_edge_attributes, node_attributes, edge_attributes
  File "/home/neba/.conda/envs/feature/lib/python3.7/site-packages/causalnex/plots/plots.py", line 197, in _add_attributes
    for node in _sm.nodes:
AttributeError: 'NoneType' object has no attribute 'nodes'
28-08-2021 15:06:25 - [ERROR] -  structure - (structure.py).drawgraph(line 17) - plots causal graph  failed.
Traceback (most recent call last):
  File "/home/neba/Desktop/Industry - Casualty Challenge/scripts/structure.py", line 12, in drawgraph
    viz = plot_structure( sm, graph_attributes={"scale": "2.0","size": "2.5"}, all_node_attributes=NODE_STYLE.WEAK, all_edge_attributes=EDGE_STYLE.WEAK)
  File "/home/neba/.conda/envs/feature/lib/python3.7/site-packages/causalnex/plots/plots.py", line 105, in plot_structure
    sm, all_node_attributes, all_edge_attributes, node_attributes, edge_attributes
  File "/home/neba/.conda/envs/feature/lib/python3.7/site-packages/causalnex/plots/plots.py", line 197, in _add_attributes
    for node in _sm.nodes:
AttributeError: 'NoneType' object has no attribute 'nodes'
28-08-2021 15:06:33 - [ERROR] -  structure - (structure.py).drawgraph(line 17) - plots causal graph  failed.
Traceback (most recent call last):
  File "/home/neba/Desktop/Industry - Casualty Challenge/scripts/structure.py", line 12, in drawgraph
    viz = plot_structure( sm, graph_attributes={"scale": "2.0","size": "2.5"}, all_node_attributes=NODE_STYLE.WEAK, all_edge_attributes=EDGE_STYLE.WEAK)
  File "/home/neba/.conda/envs/feature/lib/python3.7/site-packages/causalnex/plots/plots.py", line 105, in plot_structure
    sm, all_node_attributes, all_edge_attributes, node_attributes, edge_attributes
  File "/home/neba/.conda/envs/feature/lib/python3.7/site-packages/causalnex/plots/plots.py", line 197, in _add_attributes
    for node in _sm.nodes:
AttributeError: 'NoneType' object has no attribute 'nodes'
28-08-2021 15:06:48 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 15:08:08 - [INFO] -  structure - (structure.py).drawgraph(line 14) - plots causal graph successfully
28-08-2021 15:09:08 - [INFO] -  structure - (structure.py).drawgraph(line 14) - plots causal graph successfully
28-08-2021 15:09:19 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 15:10:21 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:11:23 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:11:42 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:15:32 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 15:15:32 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 15:17:01 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:17:01 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:21:49 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:21:49 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:22:56 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:22:56 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:24:47 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:24:47 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:25:05 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 15:26:31 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:28:05 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:28:22 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 15:28:42 - [INFO] -  file_read - (file_read.py).read_csv(line 21) - Csv file read from ../input/data.csv.
28-08-2021 15:30:01 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
28-08-2021 15:31:28 - [INFO] -  structure - (structure.py).drawgraph(line 13) - plots causal graph successfully
